torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,46 @@
+import io
+from typing import Optional
+
+import torch
+
+from .export_options import ExportOptions
+
+
+def dynamo_export(
```

Comment:
Parameter names for consistency and clarity?

```python
def dynamo_export(
    model: torch.nn.Module,
    /,
    *model_args,
    export_options: Optional[ExportOptions] = None,
    **model_kwargs
) -> ExportOutput:
    ...
```

This makes it very clear to me that the args and kwargs belong to the model and the options influence the export. Added bonus at the expense of characters is `export_options` is less likely to introduce conflict with within `**model_kwargs`.

Reply:
looks goot to me, except for the uuid lol